src/tss2-fapi/api/Fapi_CreateKey.c

```diff
@@ -179,6 +181,14 @@ Fapi_CreateKey_Async(
                             &context->cmd.Key_Create.public_templ);
     return_if_error(r, "Set key flags for key");
 
+    /* If neither sign nor decrypt is set both flags will
+       sign_encrypt and decrypt have to be set. */
+    if ( !(*attributes & TPMA_OBJECT_SIGN_ENCRYPT) &&
```

Comment:
No need for the space after '(' and before ')'. The style should be consistent with the rest.